lib/redmon.rb

```diff
@@ -27,10 +27,13 @@ def start_em
 
   def start_app
     require 'thin'
+    base_path = config.base_path
     Thin::Server.start(*config.web_interface) do
-      run Redmon::App.new
+      map base_path do
```

Comment:
It would be (I need to look to make sure) more efficient if an instance of Redmon::App was constructed outside and then referenced by the block.

```ruby
app = Redmon::App.new
map base_path do
  run app
end

or if you want to module scope the instance

map base_path do
  run @@app ||= Redmon::App.new
end

```

thoughts?